Data-Explorer/server.R

```diff
@@ -14,21 +14,19 @@
 
 ### Server ----
 
-#credentials <- readRDS("data/admin/credentials.rds")
+
 
  function(input, output, session) {
```

Comment:
same here, note might be helpful to distinguish what these different sections are doing.